schema-extended/arxiv-extended.json

```diff
@@ -0,0 +1,623 @@
+        ]
+      },
+      "unversioned_identifier": {
+        "title": "arXiv canonical identifier",
+        "type": "string",
+        "pattern": "^(([0-9]{4}\\.[0-9]{4,5})|([a-z\\-]+\\/[0-9]{2}[01][0-9]{4}))$"
+      },
+      "versioned_identifier": {
+        "title": "arXiv canonical identifier with version affix",
+        "type": "string",
+        "pattern": "^(([0-9]{4}\\.[0-9]{4,5})|([a-z\\-]+\\/[0-9]{2}[01][0-9]{4}))v[0-9]+$"
+      },
+      "version": {
+
+        "type": "object",
+        "properties": {
+          "content_types": {
+            "title": "List of content types",
+            "type": "array",
+            "items": {
+              "$ref": "#/definitions/content_type"
+            },
+            "minItems": 1,
+            "uniqueItems": true
+          },
+          "submitter": {
+            "title": "Submitter",
+            "description": "Information about the submitter and possible proxy for the submitter.",
+            "type": "object",
+            "properties": {
+              "submitter": {
+                "$ref": "#/definitions/person"
+              },
+              "proxy_name": {
+                "title": "Proxy name",
+                "description": "Name of the proxy that submitted on the submitter's behalf.",
+                "type": "string",
+                "minLength": 1
+              }
+            },
+            "required": [
+              "submitter"
+            ]
+          },
+          "submitted_date": {
+            "description": "Date this e-print version was submitted.",
+            "type": "string",
+            "format": "date-time"
+          },
+          "publication_date": {
+            "description": "Date this e-print version was announced.",
+            "type": "string",
+            "format": "date-time"
+          },
+          "license": {
+            "title": "License",
+            "description": "The license associated with the content.",
+            "oneOf": [
+              {"$ref": "#/definitions/active_licenses"},
+              {"$ref": "#/definitions/inactive_licenses"}
+            ]
+          },
+          "titles": {
+            "title": "Titles",
+            "description": "The English and non-English versions of the title.",
+            "$ref": "#/definitions/titles"
+          },
+          "abstracts": {
+            "$ref": "#/definitions/abstracts"
+          },
+          "legacy_authors": {
+            "title": "Legacy authors",
+            "description": "Legacy author free-text field.",
+            "type": "string",
+            "minLength": 1,
+            "examples": [
+              "J. Doe (Cornell University), M. Smith (Stanford University), F. Bar"
+            ]
+          },
+          "authors": {
+            "title": "Authors",
+            "type": "array",
+            "items": {
+              "$ref": "#/definitions/person"
+            },
+            "minItems": 1
+          },
+          "collaborations": {
```

Comment:
Collaborations doesn't capture what currently is encoded in arXiv. authorA, authorB (collboration X) AuthorC, authorD (collaboration Y) is understood to mean that authors A and B are from collaboration X and C and D are from collaboration Y.

But... I don't think we actually do anything with that information. So maybe this is fine?

Reply:
Indeed, this is problematic. I'm not sure how to reconcile author-level collaborations with document-level collaborations. Will need to discuss this.